tests/sentry/api/endpoints/test_event_owners.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import absolute_import
+
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase
+
+from sentry.models import ProjectOwnership
+from sentry.ownership.grammar import Rule, Owner, Matcher, dump_schema
+
+
+class ProjectOwnershipEndpointTestCase(APITestCase):
+    def setUp(self):
+        self.login_as(user=self.user)
+        self.user2 = self.create_user("user2@example.com")
+        self.user3 = self.create_user("user3@example.com")
+
+        self.team = self.create_team(
+            organization=self.organization, slug="tiger-team", members=[self.user]
+        )
+        self.team2 = self.create_team(
+            organization=self.organization, slug="tiger-team2", members=[self.user2]
+        )
+        self.team3 = self.create_team(
+            organization=self.organization, slug="tiger-team3", members=[self.user3]
+        )
+
+        self.project = self.project = self.create_project(
+            organization=self.organization, teams=[self.team, self.team2, self.team3], slug="bengal"
+        )
+
+    def test_no_rules(self):
+        event1 = self.store_event(
+            data={"stacktrace": {"frames": [{"filename": "foo.py"}]}}, project_id=self.project.id
+        )
+
+        self.path = reverse(
+            "sentry-api-0-event-owners",
+            kwargs={
+                "organization_slug": self.organization.slug,
+                "project_slug": self.project.slug,
+                "event_id": event1.event_id,
+            },
+        )
+
+        resp = self.client.get(self.path)
+        assert resp.status_code == 200
+        assert len(resp.data["owners"]) == 0
```

Comment:
Should you also assert the rule key in the response?

Reply:
Yeah I should cover those as well.